test/check-application

```diff
@@ -51,13 +51,88 @@ class TestApplication(testlib.MachineCase):
+        b.wait_visible('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-delete')
+        b.click('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-delete')
+        with b.wait_timeout(60):
+            b.wait_present(".modal-dialog div")
+        b.wait_visible(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
+        with b.wait_timeout(60):
+            try:
+                b.wait_visible("#containers-containers")
+                b.wait_not_in_text("#containers-containers", "alpine:latest")
+            except:
+                self.confirm()
+                b.wait_visible("#containers")
+                b.wait_not_in_text("#containers-containers", "alpine:latest")
+
+        # delete a running container
+        self.loadAllContainerlist()
+        b.wait_present('#containers-containers tr:contains("busybox:latest")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.wait_visible('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
+        with b.wait_timeout(60):
+            b.wait_present(".modal-dialog div")
+        b.wait_visible(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
+        with b.wait_timeout(60):
+                self.confirm()
+                with b.wait_timeout(60):
+                    b.wait_not_in_text("#containers-containers", "busybox:latest")
+
+        # run a container for Stop test
+        m.execute("podman run -d --name swamped-crate busybox sleep 1000")
+        self.loadAllContainerlist()
+        b.wait_present('#containers-containers tr:contains("busybox:latest")')
+        b.wait_visible('#containers-containers tr:contains("busybox:latest")')
+
+        # stop a running container
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+        b.wait_visible('#containers-containers tbody tr:contains("busybox:latest") + tr:contains("Stop")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") + tr:contains("Stop")')
+        self.loadAllContainerlist()
+        # TODO: wait for container to stop
+        b.wait_present('#containers-containers tr:contains("busybox:latest")')
+        b.wait_visible('#containers-containers tr:contains("busybox:latest")')
+        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
+
+    # FIXME: UI needs to listen to change signals and update automatically
+
+    def confirm(self):
+        b = self.browser
+        b.wait_present(".modal-dialog div")
+        b.wait_visible(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
+
+    def loadAllContainerlist(self):
```

Comment:
This should have a docstring that this includes the `reload()` hack. I. e. it should only be used after running a CLI command, not after UI actions.